ui/app/AppLayouts/Wallet/controls/ManageTokenMenuButton.qml

```diff
@@ -118,20 +119,58 @@ StatusFlatButton {
                 }
             }
 
+            // (hide) collection tokens
+            StatusMenu {
+                id: collectionSubmenu
+                enabled: !root.inHidden && root.isCollectible && !root.isCommunityAsset && !root.isGroup
+                title: qsTr("Hide")
+                assetSettings.name: "hide"
+                type: StatusAction.Type.Danger
+
+                StatusAction {
+                    objectName: "miHideCollectionToken"
+                    text: qsTr("This collectible")
+                    onTriggered: {
+                        root.showHideRequested(model.symbol, false)
+                        communitySubmenu.dismiss()
+                    }
+                }
+                StatusAction {
+                    objectName: "miHideAllCollectionTokens"
+                    text: qsTr("All collectibles from this collection")
+                    onTriggered: {
+                        root.showHideGroupRequested(root.groupId, false)
+                        communitySubmenu.dismiss()
+                    }
+                }
+            }
+
             // token group
             StatusAction {
                 objectName: "miHideTokenGroup"
                 enabled: !root.inHidden && root.isGroup
                 type: StatusAction.Type.Danger
                 icon.name: "hide"
-                text: root.isCollectible ? qsTr("Hide all collectibles from this community") : qsTr("Hide all assets from this community")
+                text: {
+                    if (root.isCollection) {
+                        return root.isCollectible ? qsTr("Hide all collectibles from this collection") : qsTr("Hide all assets from this collection")
```

Comment:
I guess here we can keep the `root.isCollectible` check but `assets` text will never happen (doesn't make sense) since we don't have collections while working with assets.

Reply:
Removed